plugins/minigames.json

```diff
@@ -919,17 +919,23 @@
         }
       }
     },
-    "super_duel": {
+    "duel": {
```

Comment:
renaming filenames of any plugin/minigame is supposed to be a breaking change, so let's do this if there feels a good need to do so

to make renaming work, we'll have to remove any previous versions from metadata file, as the previous version of this minigame had the name `super_duel` whose entry won't be found in `minigames.json` any more